spec/mailers/shf_application_mailer_spec.rb

```diff
@@ -125,26 +125,6 @@ def make_branding_payment(a_company)
         expect(email_sent).to have_body_text(I18n.t('mailers.shf_application_mailer.acknowledge_received.your_submitted_info'))
     end
 
-
-    describe 'shows text We will do the review after we receive your files only if no files are uploaded' do
-
-      it 'no files uploaded' do
-        expect(email_sent).to have_body_text(I18n.t('mailers.shf_application_mailer.acknowledge_received.review_after_recevied_files'))
-      end
-
-      it 'a file uploaded (text not shown)' do
-        fn = file_fixture('image.gif')
-        received_app.uploaded_files.create(actual_file: File.open(fn, 'r'))
-        expect(email_sent).not_to have_body_text(I18n.t('mailers.shf_application_mailer.acknowledge_received.review_after_recevied_files'))
-        received_app.uploaded_files.delete_all
-      end
-
-    end
-
-    it_behaves_like 'it shows how to login and the page to upload files' do
```

Comment:
This test should stay [lines 144 - 146].  Don't delete it.  (This text is still in the email; we need to verify that it is there.)